app/application.js

```diff
@@ -27,80 +27,39 @@ const WINDOW_MIN_HEIGHT = 400;
 export const Platforms = {
   Mac: 1,
   Windows: 2,
-  Linux: 3,
-  isMac: (platform) => {
-    return platform === Platforms.Mac;
-  },
+  Linux: 3
 };
 
-export class DesktopApplication {
-
-  static instance = null;
-
-  static createSharedApplication() {
-    const getPlatform = () => {
-      const processPlatform = process.platform;
-      if (processPlatform === 'darwin') {
-        return Platforms.Mac;
-      } else if (processPlatform === 'win32') {
-        return Platforms.Windows;
-      } else if (processPlatform === 'linux') {
-        return Platforms.Linux;
-      }
-    };
-    this.instance = new DesktopApplication({
-      platform: getPlatform()
-    });
+function determinePlatform(platform) {
+  switch (platform) {
+    case "darwin":
+      return Platforms.Mac;
+    case "win32":
```

Comment:
Should be single quotes here

Reply:
Probably good to add this rule to eslint: https://eslint.org/docs/rules/quotes.html

Reply:
A good reference: https://standardjs.com/rules.html